medmodels/medrecord/medrecord.py

```diff
@@ -1234,6 +1234,17 @@ def __getitem__(
 
         return self.select_edges(key)
 
+    def clone(self) -> "MedRecord":
```

Comment:
Why "MedRecord" and not MedRecord? Circular import workaround? In all of our modules we give as return the name of the class without the quotation marks so it would be nice to have a standard for it!

Reply:
Idk why :D
Removed